deploy/20_register_templates.ts

```diff
@@ -124,57 +124,72 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
     hre.deployments.log('"HolographERC721" is already registered');
   }
 
-  // Get the contracts rquired for the ERC721Drop
-  const EditionMetadataRenderer = await hre.deployments.get('EditionMetadataRenderer');
-  const HolographFeeManager = await hre.deployments.get('HolographFeeManager');
-
-  // Register ERC721Drop
-  const futureErc721DropAddress = await genesisDeriveFutureAddress(
+  // Register HolographDropsEditionsV1
+  const futureEditionsMetadataRendererAddress = await genesisDeriveFutureAddress(
     hre,
     salt,
-    'HolographERC721Drop',
-    generateInitCode(
-      ['tuple(address,address,address,string,string,address,address,uint64,uint16,bytes[],address,bytes)', 'bool'],
+    'EditionsMetadataRenderer',
+    generateInitCode([], [])
+  );
+  const futureEditionsMetadataRendererProxyAddress = await genesisDeriveFutureAddress(
+    hre,
+    salt,
+    'EditionsMetadataRendererProxy',
+    generateInitCode(['address', 'bytes'], [futureEditionsMetadataRendererAddress, generateInitCode([], [])])
+  );
+  const holographDropsEditionsV1InitCode = generateInitCode(
+    [
+      'tuple(address,address,address,address,uint64,uint16,tuple(uint104,uint32,uint64,uint64,uint64,uint64,bytes32),address,bytes)',
+    ],
+    [
       [
-        [
-          HolographFeeManager.address, // holographFeeManager
-          '0x0000000000000000000000000000000000000000', // holographERC721TransferHelper
-          '0x000000000000AAeB6D7670E522A718067333cd4E', // marketFilterAddress (opensea)
-          'Holograph ERC721 Drop Collection', // contractName
-          'hDROP', // contractSymbol
-          deployer.address, // initialOwner
-          deployer.address, // fundsRecipient
-          1000, // 1000 editions
-          1000, // 10% royalty
-          [], // setupCalls
-          EditionMetadataRenderer.address, // metadataRenderer
-          generateInitCode(['string', 'string', 'string'], ['decscription', 'imageURI', 'animationURI']), // metadataRendererInit
-        ],
-        true, // skipInit
-      ]
-    ) // initCode
+        '0x0000000000000000000000000000000000000000', // holographERC721TransferHelper
+        '0x0000000000000000000000000000000000000000', // marketFilterAddress (opensea)
+        deployer.address, // initialOwner
+        deployer.address, // fundsRecipient
+        0, // 1000 editions
+        1000, // 10% royalty
+        [0, 0, 0, 0, 0, 0, '0x' + '00'.repeat(32)], // setupCalls
```

Comment:
What's happening here? Is it all the params for sales config?

Reply:
Yep this is sales config data. Again it’s empty cuz this is deployment of source where no actual data is being used.

Reply:
got it 👍